### ISSUE TITLE
t/01-basic.t may fail

### ISSUE THREAD

**Issue by eserte** [perlancar/perl-Text-Table-More#3]
On some of my systems the test suite fails:
```
    #   Failed test at t/01-basic.t line 376.
    #          got: '| A |
    # `---'
    # '
    #     expected: '.---.
    # | A |
    # `---'
    # '
    # expected:
    # .---.
    # | A |
    # `---'
    # 
    # result:
    # | A |
    # `---'
    # Looks like you failed 1 test of 1.

#   Failed test '1x1'
#   at t/01-basic.t line 378.

    #   Failed test at t/01-basic.t line 376.
    #          got: '| A | BBB |
    # `---+-----'
    # '
    #     expected: '.---+-----.
    # | A | BBB |
    # `---+-----'
    # '
    # expected:
    # .---+-----.
    # | A | BBB |
    # `---+-----'
    # 
    # result:
    # | A | BBB |
    # `---+-----'
    # Looks like you failed 1 test of 1.

#   Failed test '1x2'
#   at t/01-basic.t line 378.

    #   Failed test at t/01-basic.t line 376.
    #          got: '| A   |
    # | BBB |
    # `-----'
    # '
    #     expected: '.-----.
    # | A   |
    # | BBB |
    # `-----'
    # '
    # expected:
    # .-----.
    # | A   |
    # | BBB |
    # `-----'
    # 
    # result:
    # | A   |
    # | BBB |
    # `-----'
    # Looks like you failed 1 test of 1.

#   Failed test '2x1'
#   at t/01-basic.t line 378.

    #   Failed test at t/01-basic.t line 376.
    #          got: '| A  | BBB |
    # | CC | D   |
    # `----+-----'
    # '
    #     expected: '.----+-----.
    # | A  | BBB |
    # | CC | D   |
    # `----+-----'
    # '
    # expected:
    # .----+-----.
    # | A  | BBB |
    # | CC | D   |
    # `----+-----'
    # 
    # result:
    # | A  | BBB |
    # | CC | D   |
    # `----+-----'
    # Looks like you failed 1 test of 1.

#   Failed test '2x2'
#   at t/01-basic.t line 378.
# Looks like you failed 4 tests of 17.
t/01-basic.t ................. 
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/17 subtests 
```

Statistical analysis does not give any clues, and the fail matrix looks quite random: http://matrix.cpantesters.org/?dist=Text-Table-More%200.014

**Comment by perlancar** [comment]
Currently can't reproduce and can't think of the culprit for the first line being eaten, marking as [stalled].

**Comment by eserte** [comment]
Can be reproduced if an older BorderStyleBase.pm is installed. Create a Dockerfile with the following contents:
```
FROM debian:10

RUN apt-get -y update
RUN apt-get -y --no-install-recommends install perl make gcc cpanminus libc6-dev
RUN cpanm --notest BorderStyleBase@0.003
CMD cpanm --test PERLANCAR/Text-Table-More-0.014.tar.gz || (cat /root/.cpanm/work/*/build.log; false)
```
And run
```
docker build -t perl-test . && docker run -it perl-test
```

**Comment by perlancar** [comment]
Got it, fixed in ed0607c. Thanks.
